src/app/main.module.ts

```diff
@@ -3,11 +3,13 @@ import { TaskModule } from 'src/app/task/task.module';
 import { StandupModule } from './standup/standup.module';
 import { SubscriptionModule } from './subscription/subscription.module';
 import { SummaryModule } from './summary/summary.module';
+import { WorkspaceUsersModule } from './workspace/workspace-users/workspace-users.module';
 import { WorkspaceModule } from './workspace/workspace.module';
 
 @Module({
   imports: [
     WorkspaceModule,
+    WorkspaceUsersModule,
```

Comment:
this module probably need to be imported in workspace module and better name this module UserWorkspaceModule like we hava AdminWorkspaceModule to be consistent with naming